roles/edpm_libvirt/templates/auth.conf

```diff
@@ -0,0 +1,8 @@
+{% if edpm_libvirt_sasl_auth_enabled|bool %}
+[credentials-overcloud]
+authname=migration@overcloud
```

Comment:
again we do not use the term overcloud here so we should use a different value here.

based on the context i would suggest nova or OpenStack.

Reply:
copied from puppet-tripleo, didn't want to change it in case it's significant for adoption (probably not)

Reply:
can address nits such as this in a follow-up

Reply:
maybe significant actually, if client sends authname to server. Not willing to risk it for now, can return to it once there is CI coverage for adoption with TLS

Reply:
.. and while Gibi (IIRC) mentioned the intention is to not support live-migration during adoption, I think it would be a good thing to avoid breaking it

Reply:
There is no way to do VM lifecycle operations during adoption as at the start of the adoption we stop the nova-compute service on each hypervisor.

Reply:
yep during adoption this code will run and regenerate all the auth config so there should be no impact on adoption to using a more approate name.